test/unit/Validation/ValidatorTest.php

```diff
@@ -22,6 +22,20 @@ public function createDependencies(): void
         $this->token = $this->createMock(Token::class);
     }
 
+    /**
+     * @test
+     *
+     * @covers \Lcobucci\JWT\Validation\Validator::assert
+     */
+    public function assertShouldRaiseExceptionWhenNoConstraintIsGiven(): void
+    {
+        $validator = new Validator();
+
+        $this->expectException(NoContraintGiven::class);
+
+        $validator->assert($this->token, ...[]);
```

Comment:
We can simply omit the second argument, makes things simpler here :+1:

Reply:
Actually I made it intentional.

It needs to be obvious to anyone that here I'm passing something, empty but something.

If someone would ever refactor `assert` method with a new syntax or type hint, we must prevent he/she to look at this test and think "Oh, this test doesn't provide the second parameter, but now it is required by code/tool/whatever, so I can delete it".

I don't know if I'm clear here: the test shouldn't be simple; the test should express the intention.

Reply:
Fair point :+1: